apps/backend/src/services/polling_service.py

```
@@ -262,66 +332,84 @@ async def _collect_drive_health(self, device: Device) -> None:
+                        lines = smart_result.stdout.split("\n")
+                        for line in lines:
+                            if (
+                                "Temperature_Celsius" in line
+                                or "Airflow_Temperature_Cel" in line
+                            ):
+                                parts = line.split()
+                                if len(parts) >= 10:
+                                    try:
+                                        temperature = int(parts[9])
+                                    except (ValueError, IndexError):
+                                        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ValueError, IndexError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError, IndexError)`

(SIM105)